build.gradle

```diff
@@ -1,14 +1,15 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+    ext.kotlin_version = '1.3.72'
```

Comment:
might want to consider using kotlin gradle files, makes dealing with variables so much easier but minor considering how small we expect there gradle files to be

Reply:
I'll look into it, it might be worth doing but I haven't used Kotlin gradle files before, always stuck with basic Gradle.

Reply:
But it's probably something I'd convert to in its own PR